workers/task-driver/integration/main.rs

```diff
@@ -181,6 +182,9 @@ fn setup_arbitrum_client_mock(test_args: &CliArgs) -> ArbitrumClient {
 
 /// Setup code for the integration tests
 fn setup_integration_tests(test_args: &CliArgs) {
+    // Setup token remaps, this is used in metrics collection, will panic if not set
+    TOKEN_REMAPS.set(Default::default()).unwrap();
```

Comment:
hmm, should we gate metrics collection behind a cargo feature? the current mentality is to defer all configuration of the node to the config file, but this seems annoying (and maybe other users will want to completely disable for perf reasons)

Reply:
I think this is fine for now, no need to complicate things